src/main/kotlin/dev/arbjerg/ukulele/command/RepeatCommand.kt

```diff
@@ -0,0 +1,33 @@
+package dev.arbjerg.ukulele.command
+
+import dev.arbjerg.ukulele.features.HelpContext
+import dev.arbjerg.ukulele.jda.Command
+import dev.arbjerg.ukulele.jda.CommandContext
+import org.springframework.stereotype.Component
+
+/**
+ * A Repeat command that is only 'Repeat One' right now.  In the future,
+ * it can be expanded to have a 'Repeat All'.
+ */
+@Component
+class RepeatCommand : Command ("repeat", "r", "loop") {
+    override suspend fun CommandContext.invoke() {
+        if (argumentText.isBlank()) {
+            player.toggleRepeat()
+        } else {
+            player.repeatOne = argumentText.toBoolean()
+        }
```

Comment:
So with no args it toggles the repeatOne flag.  With 'true' it enabled Repeat One.  With 'false' (or anything else) it disables Repeat One.